LunaBot/Commands/User/SnugCommand.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Discord.WebSocket;
+using LunaBot.Database;
+
+namespace LunaBot.Commands
+{
+    [LunaBotCommand("Snug","snug")]
```

Comment:
caps don't matter for commands, they all get set to lowered case. You could change the second snug into `s` to keep the design of shorter commands.